lib/src/runner/configuration/args.dart

```diff
@@ -92,6 +92,11 @@ final ArgParser _parser = (() {
           'to provide improved test performance but at the cost of\n'
           'debuggability.',
       defaultsTo: true);
+  parser.addFlag("retry",
+      help: 'Retry failing tests if configured to do so.\n'
+          'Useful for iterating locally on a failing test with retry '
+          'configuration.',
```

Comment:
I don't think this second line pulls its weight. Every extra lines in the `--help` output costs a lot in terms of visual complexity.